samples/unit-tests/series-flags/general/demo.js

```diff
@@ -165,3 +165,53 @@ QUnit.test('Flags visibility', function (assert) {
         'Flag with box position value equal to 0 is visible.'
     );
 });
+
+// The flag series should be invertible (#14063).
```

Comment:
Should it? 
Comment in the `FlagSeries.ts` says that it shouldn't be invertible.